stdlib/2/os/__init__.pyi

```diff
@@ -197,18 +197,20 @@ def walk(top: AnyStr, topdown: bool = ..., onerror: Any = ...,
                                                     List[AnyStr]]]: ...
 
 def abort() -> None: ...
-def execl(file: _PathType, *args) -> None: ...
-def execle(file: _PathType, *args) -> None: ...
-def execlp(file: _PathType, *args) -> None: ...
-def execlpe(file: _PathType, *args) -> None: ...
+def execl(path: _PathType, arg0: Union[bytes, Text], *args: Union[bytes, Text]) -> NoReturn: ...
```

Comment:
I don't think we need `arg0` here—the function is implemented as just taking args.